applications/nrf_desktop/doc/profiler_sync.rst

```diff
@@ -3,12 +3,12 @@
 Profiler synchronization module
 ###############################
 
-Use the profiler synchronization module to synchronize timestamps of :ref:`profiler` events for two devices connected over physical wire.
-Timestamp synchronization is required to increase accuracy of measured times between the profiler events that come from two different devices.
+Use the profiler synchronization module to synchronize the timestamps of :ref:`profiler` events between two devices connected over a physical wire.
+Timestamp synchronization is required to increase the accuracy of the measured times between the profiler events that come from two different devices.
 
 The profiler data is collected separately from both devices using dedicated Python scripts.
-Then the data can be merged together using the ``merge_data.py`` script with ``sync_event`` used as synchronization event for both peripheral and central.
-For more detailed information, see :ref:`profiler` documentation.
+The data can then be merged using the ``merge_data.py`` script with ``sync_event`` used as a synchronization event for both peripheral and central.
```

Comment:
Used ``Peripheral`` and ``Central`` to be consistent in the module documentation page.